en_US/network/emqx-mqtt-tls.md

```diff
@@ -91,6 +91,60 @@ listeners.ssl.default {
 
 If the `RSA-PSK` cipher suites are used, the `RSA` certificate is still required, see [RFC4279](https://www.rfc-editor.org/rfc/rfc4279#section-4) for details.
 
+### OCSP Stapling
+
+Since EMQX version v5.0.23 and e5.0.3, Online Certificate Status Protocol (OCSP) Stapling is supported for MQTT SSL listeners.  Note that those do not include Secure WebSocket nor QUIC listeners: only listeners of type `ssl` support this feature.
```

Comment:
Note to self: check that those are the official versions where the features will be released indeed.